app/controllers/quotes_controller.rb

```diff
@@ -1,7 +1,6 @@
 class QuotesController < ApplicationController
-
   def index
     @quotes = Quote.all
   end
 
-end
\ No newline at end of file
+end
```

Comment:
Style-nit: our styleguide says that there should not be whitespaces around the class declaration -- and this change seems to be style only (not functional) can you please revert these changes?